esmvaltool/cmorizers/data/formatters/datasets/jma_transcom.py

```diff
@@ -14,7 +14,6 @@
     (Department of Atmosphere, Ocean and Earth System Modeling Research,
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Fixed